libr/anal/meta.c

```diff
@@ -626,12 +626,17 @@ R_API void r_meta_print(RAnal *a, RAnalMetaItem *d, int rad, PJ *pj, bool show_f
 			pj_kn (pj, "offset", d->from);
 			pj_ks (pj, "type", r_meta_type_to_string (d->type));
 
-			pj_k (pj, "name");
-			if (d->type == 's' && (base64_str = r_base64_encode_dyn (d->str, -1))) {
+			pj_k (pj, "color");
```

Comment:
do only this on highlight metas, otherwise you are breaking comments and many other metadatas. see the XX] tests in travis logs

Reply:
Ah..yes i had that question but you didn't mention in review so i thought we have to change it to color.